common/utils/EventLogRiemannClient.js

```diff
@@ -104,6 +105,11 @@ class EventLogRiemannClient {
       if (appGuid !== undefined) {
         eventName = `${eventName}.app_guid.${appGuid}`;
       }
+
+      if (serviceId !== undefined) {
+        let serviceName = require('../models').catalog.getServiceName(serviceId);
+        eventName = `${eventName}.service_name.${serviceName}`;
```

Comment:
Would this not impact few of the existing event names? Reason to ask this Q is, i guess we have rules set already based on the above pattern right, now if we are changing this should we also be changing those Riemann pattern? 

Also `service_name` in suffix seems redundant, as the actual service name itself will be self-explanatory, in case of guids it makes sense but in case of explicit name, this might be redundant. Again naming is always subjective, so your final call on naming.

Reply:
Would this not impact few of the existing event names? --> yes

Reason to ask this Q is, i guess we have rules set already based on the above pattern right, now if we are changing this should we also be changing those Riemann pattern? --> yes

Also service_name in suffix seems redundant, as the actual service name itself will be self-explanatory, in case of guids it makes sense but in case of explicit name, this might be redundant. Again naming is always subjective, so your final call on naming. --> yes , agree with you . i had the same thought , maybe even to have it earlier in the event name. But kept it this way to follow the same convention for all the additional info suffixed to event name.

Reply:
ok